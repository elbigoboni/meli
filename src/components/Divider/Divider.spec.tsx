import { render } from "@testing-library/react";
import React from "react";
import { Divider } from ".";

describe("<Divider />", () => {
  describe("Render", () => {
    test("should render correctly", () => {
      const { container } = render(<Divider />);
      expect(container).toMatchSnapshot();
    });
  });
});
