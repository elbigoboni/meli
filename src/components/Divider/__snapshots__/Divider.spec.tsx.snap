// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> Render should render correctly 1`] = `
<div>
  <hr
    class="sc-bdfBwQ iAZiBa"
  />
</div>
`;
